src/css/SignInScreen.css

```diff
@@ -0,0 +1,27 @@
+.signInScreen {
```

Comment:
Instead of using this css file, let's use styled components

Reply:
https://www.styled-components.com/

Reply:
Done